modules/core/src/main/java/org/apache/ignite/internal/commandline/tracing/configuration/TracingConfigurationArguments.java

```diff
@@ -0,0 +1,178 @@
+/*
+ * Copyright 2020 GridGain Systems, Inc. and Contributors.
+ *
+ * Licensed under the GridGain Community Edition License (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.gridgain.com/products/software/community-edition/gridgain-community-edition-license
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.ignite.internal.commandline.tracing.configuration;
+
+import java.util.Collections;
+import java.util.Set;
+import org.apache.ignite.internal.processors.tracing.Scope;
+import org.apache.ignite.internal.processors.tracing.Span;
+import org.apache.ignite.internal.visor.tracing.configuration.VisorTracingConfigurationItem;
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+
+/**
+ * This class contains all possible arguments after parsing tracing-configuration command input.
+ */
+ public final class TracingConfigurationArguments extends VisorTracingConfigurationItem {
+    /** */
+    private static final long serialVersionUID = 0L;
+
+    /** Command. */
+    private TracingConfigurationSubcommand cmd;
+
+    /**
+     * Default constructor.
+     */
+    public TracingConfigurationArguments() {
+        // No-op.
+    }
+
+    /**
+     * Constructor.
+     *
+     * @param cmd Command
+     * @param scope Specify the {@link Scope} of a trace's root span
+     *  to which some specific tracing configuration will be applied.
+     * @param lb Specifies the label of a traced operation. It's an optional attribute.
+     * @param samplingRate Number between 0 and 1 that more or less reflects the probability of sampling specific trace.
+     *  0 and 1 have special meaning here, 0 means never 1 means always. Default value is 0 (never).
+     * @param supportedScopes Set of {@link Scope} that defines which sub-traces will be included in given trace.
+     *  In other words, if child's span scope is equals to parent's scope
+     *  or it belongs to the parent's span supported scopes, then given child span will be attached to the current trace,
+     *  otherwise it'll be skipped.
+     *  See {@link Span#isChainable(org.apache.ignite.internal.processors.tracing.Scope)} for more details.
+     */
+    public TracingConfigurationArguments(
+        TracingConfigurationSubcommand cmd,
+        Scope scope,
+        String lb,
+        double samplingRate,
+        Set<Scope> supportedScopes)
```

Comment:
Minor: please take a look at https://cwiki.apache.org/confluence/display/IGNITE/Coding+Guidelines#CodingGuidelines-MethodArguments
I think it should be:
```
    public TracingConfigurationArguments(
        ...
        Set<Scope> supportedScopes
    ) {
```

Reply:
Fixed.